### PR TITLE
Support Add sparse arrays

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1375

### Summary of Changes

* adds an example
* lets see if we can add the windows fix here too

### CODE REVIEW

File: src/test/examples/sparse-array.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  <body>
+    <script>
+      debugger;
+
+      localStorage[1] = "a test";
+      localStorage[2] = "a";
+      localStorage[66261280] = "another test";
+      let data = JSON.parse(JSON.stringify(localStorage));
+
```
Comment (bjorn3) [inline src/test/examples/sparse-array.html]: Debugger statement should be put here. (if data doesnt have a value it wont crash)

File: src/test/examples/sparse-array.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      console.log(data) // Put breakpoint here
```
Comment (bjorn3) [inline src/test/examples/sparse-array.html]: This line could be removed.

Comment (bjorn3) [inline src/test/examples/sparse-array.html]: Except if the js engine may optimize `data` away.

**Comment by jasonLaster** [comment]
moving over here
https://github.com/jasonLaster/debugger-examples

**Pull request closed without merging**